app/javascript/vite/application.js

```diff
@@ -61,7 +60,10 @@ document.addEventListener('DOMContentLoaded', () => {
     })
   }
 
-  setTimeout(checkMissingAuthToken, 1000);
+  if (window.SCP.userSignedIn) {
+    ScpApi.setUpRenewalForUserAccessToken()
+  }
```

Comment:
Blocker: I'm concerned that we silently drop a clause from the [original check](https://github.com/broadinstitute/single_cell_portal_core/pull/1845/files#diff-154a7bd91be253d5383f8b76ec079079cff01ce9b80c97a633ebbaae1b5e47b3L3), which was:

```
if (window.SCP.userAccessToken === '' && window.SCP.userSignedIn) {
```

Two pre-existing problems:

1.`window.SCP.userAccessToken` is an empty string if the user stays signed in but loads a new page after only an hour.  Proper handling would be to refresh the user access token server-side, _before_ the very first HTTP response in the page load (i.e., before the HTML document is returned).

2. When `window.SCP.userAccessToken` is an empty string, stuff still seems to basically work.  This is unsettling, and a sign we don't understand something important.  SCP API calls ought to fail when `window.SCP.userAccessToken === '' && window.SCP.userSignedIn`.  The fact that SCP API calls work when the bearer token is empty means that token isn't working like we expect, which is a problem.

Fixing those underlying issues is out-of-scope here.  However, this PR degrades our observability into this issue.  Before, we developers would at least get annoyed by the modal telling us to refresh the page.  We could dismiss the modal and use the page with SCP API working, but it was still a valid warning that _something is awry_.  We knew when those 2 pre-existing problems manifested.  Now, we lack even that informal observability.  We'll now only be able to detect cases of that problematic `window.SCP.userAccessToken === ''` scenario via actively querying DevTools Console.

If we don't fix problem #1 here -- and I think that'd be reasonable as a separate task -- then we at least need to not degrade our observability into the incidence of pre-existing problem #2.  I could go either way on keeping or removing the modal -- it's very annoying, but seems to report a deep true positive.  In either case, I think we need to log to Mixpanel pages that load and have `window.SCP.userAccessToken === '' && window.SCP.userSignedIn`.

Perhaps:

```suggestion
  if (window.SCP.userSignedIn) {
    ScpApi.setUpRenewalForUserAccessToken()
    if (window.SCP.userAccessToken === '') {
       log('error', 'user-access-token-empty-string')
    }
  }
```

Reply:
Good call, done in [4333211](https://github.com/broadinstitute/single_cell_portal_core/pull/1845/commits/4333211d2e96cff20c982cdc2b44a646acb645db)